tests/rows/data/rows.t2.0.sql

```diff
@@ -0,0 +1,109 @@
+("\tB>+a?]m\\9T!{2X@I:'k"),
+('8_/GI}Oh&|YPo0k,@[FB'),
+("Lnx3WawtgN'.J)\n[C-V!"),
+("$cislLng'V/`Ce[\\\nA4K"),
+('&)Jd_tHBY-!Lz?Ij0f\t*'),
+('B1KGE,#[L:@Pq8\n_*ANF'),
+("([\\c:'>U/)wDN4zQf\tC6"),
+('$G\\pS,%BRzsCUqWr^`}3'),
+('L5mNTib.KEJCRcl}>\res'),
+("MaQ.3K'e_ 7nirjET;x\r"),
+(']!62nVsu <y8vIzQYD0P'),
+('\\bWPzU(KFfmx~2E54k!A'),
+('H$<j?)C; t2S=\rEdZyL,'),
+('t\r''X)G]DLIuH$f@b"-;S'),
+('~dptv(IQ _cR7xYE1`yz'),
+('E(}|qA%-]u{)BQ:jGLVC'),
+('Rg&ya"|w60< KYJ#ElU7'),
+('N}p9uE;,rTne^0B"yLhq'),
+('{2T7!>s&@|A_<%O1v*8W'),
+('+[2lwQ!\tL{RsnK]$%E&?'),
+('h]e*:;$k"VcHi\t6%X\nQB'),
+('|<KrxRV\ri!p*FfS3:n=\t'),
+('/#f-K$Uj3y6&:i`Rh[~}'),
+('08ci<#NP:~-,[D\ry}>=A'),
+(";]KwM~F$z.(L'S-N@7Ug"),
+('IlDa4YQ[\r;?h65/ZU+mS'),
+("QwfKAPBs=MW,\r0E'i?V1"),
+("k*90\\Re!#vfYGx:)g'r3"),
+("YS:I(/<NQf>ro'`kzaH\t"),
+('1`Bq|a9;o"Y]3bM4W_Sh'),
+(".MgK'p)+@hY[L:63J>nl"),
+('h8gLDqZX:B4OTuYRV=,\\'),
+("=]W7N(1}\nKM\\m'xR[?pB"),
+('7)_p.;2xoUflRWC<"g/T'),
+('.4l]KZ?O8Vkj{;wb0JQ#'),
+("x>eRCk?(yJF\t3'g@\rzDl"),
+('4b3J{^_p7|L& jGKla\\h'),
+("a9\tobE$MG@>.[:_~c2W'"),
+('FwK`^GQ>t 3T]vh?A2c\n'),
+('7\r%]}Ypk,2LVGW:?d10f'),
+('[wmEMFr.bsU;(oflk^7@'),
+('?Kb;WeyDh3RF@\nZN,79O'),
+('?7]fDsbrUp1I|Yu5(.)N'),
+('\tH:`BZi/fbxWnA|.y~{*'),
+('md}O[U,FsQI- A1rn^b5'),
+('Fz~;,W!3_\th}c"m.V #>'),
+('w3ql-W&r*9I@R$\rHu8c '),
+('m0!FSH\\pf8GOnN &;CD>'),
+(']%-ICWRGJZc\t.BrTKLeS'),
+('pHF!GAjkau%N~[h$b7nX'),
+(' #{$TiS9w)WCmQ,}n!kv'),
+("LWR4qGO/D'&%l`3\nbe^N"),
+('S!rfCOj)y\t\r7:nF2shXK'),
+('p]de%o\tn 9|6f7IDB`V='),
+(':F^#IA%p!w t2m-U;6[E'),
+("uioc\rdDV|t@z$7'+0\nIm"),
+('7OR\t2?q]{6[QD#kCMig5'),
+('GF-LA{&> 0a)tTkO4Mj_'),
+('&Ea),-@<gF{21s7X/p"u'),
+('5=,YreMA2-y\\3+h^pJsw'),
+('\tn"d:bG$lHYQ_Dfp5)?4'),
+("o\\7!'&E}gef~M-)$Vbv>"),
+('cl5AUQHpP"+r\tj}dg)#*'),
+('S1yA;V*E9GsdT(8Hvk|e'),
+('Q.eHaOSnkX"x|]w3:;U~'),
+('%SRh2]B\nEbu> `\\wp05('),
+('1rJc8<(`2q?7sPHzxh.4'),
+('( ymTM_+QeZH6KlOR{Di'),
+("]Q'2{4iD^,~M0&K*x:<}"),
+('O"my6Egs|z$abwSA''}5 '),
+('Gl0}W[~N5: 1n=@TJ_F,'),
+('|orp1hJXz6]sZiA9DSnH'),
+('I<aZs|oK15:x6Vt^9Ww)'),
+('S/!_JAI#}`Vc\\4d5{81?'),
+('"IB[Q~L@3:c-.O{rkZ}E'),
+('8*\n%){GwO~yT^mL/ph0t'),
+('`6+/Zy.B_Y[evD@ 9zbp'),
+('%zF5&r\\^|N(*kfg{QRjS'),
+('p8y"t1-#rq:|mjhM\tWi+'),
+('=eBH%l\rP_+qUr)NYDhv;'),
+('hr\\z>-:,u+FK32qf0TXw'),
+('-sQD>=rN|[ZLR\r\tPGW6u'),
+('M)[2EZ.SyzV4dR!~\r"w9'),
+('3`Ms\t%!?zZR\no>xG|&JP'),
+(':UR8;-x2"{e3P# mlvTG'),
+("Z#|> :~or1XS*^$c'j?2"),
+('=^?D_Xf\tQ{P%UoYl<pA,'),
+('b8cQ0S\n\\>.v_usEO{J\ty'),
+('YmW$pkSLqjGxV+)A"KCP'),
+('K+aZU\\EnV!qm}vW\rd.c\t'),
+("'?dR3JHn(~)p}Gq4z-F!"),
+('\\IW(ip!2_Ab}ydfXQ01P'),
+(')#b\n&,WwYT{vyMK]\ra/\\'),
+('\\F0d~2[K/fk>Ij{^]gLH'),
+('78cm$y#4?S;PV]\\Oex=E'),
+('d"h=\n\\k)8!:C?qKD(NB['),
+(' Vemr1.kpU@B|qQ=M{O#');
+--- add unsigned primary key
+insert into `rows`.`t2` (id,a) values
+(18446744073709551605,'vr/g 8uoZ&26U+*%|(k3'),
+(18446744073709551606,':v3uo7whT`Zz>tD=[,)\r');
+
+-- analyze table for making sure the estimateCount is correct
+analyze table rows.t;
+analyze table rows.t2;
```

Comment:
Suggest putting this part into `run.sh`, not here. Let `rows.t2.0.sql` only have data.